std/json.d

```diff
@@ -1239,35 +1353,53 @@ string toJSON(const ref JSONValue root, in bool pretty = false, in JSONOptions o
                     putCharAndEOL('{');
                     bool first = true;
 
-                    void emit(R)(R names)
+                    static if (Policy.preserveMemberOrder)
                     {
-                        foreach (name; names)
+                        foreach (member; obj.members)
```

Comment:
foreach (key, value; obj.members) {...}

Reply:
Actually, in the DefaultPolicy case, `names` is just an array of strings, not an associative array.